include/jemalloc/internal/cache_bin.h

```diff
@@ -183,7 +183,8 @@ cache_bin_array_descriptor_init(cache_bin_array_descriptor_t *descriptor,
 }
 
 JEMALLOC_ALWAYS_INLINE void *
-cache_bin_alloc_easy(cache_bin_t *bin, bool *success, szind_t ind) {
+cache_bin_alloc_easy_impl(cache_bin_t *bin, bool *success, szind_t ind,
+    bool fast) {
```

Comment:
Let make the last param const, to make it explicit. How about call it “adjust_low_water”? Because when it’s false, we never modify low water.